src/models/Brand.ts

```diff
@@ -0,0 +1,19 @@
+import mongoose from 'mongoose';
+
+export type IBrandUpdate = Partial<IBrand>;
+
+export interface IBrand {
+    logoImgUrl: string;
+    backgroundImgUrl: string;
+    poolAddress: string;
+    clientId: string;
+}
+
+const brandSchema = new mongoose.Schema({
+    logoImgUrl: String,
+    backgroundImgUrl: String,
+    poolAddress: String,
+    clientId: String,
```

Comment:
@GarfDev What are we exactly doing with the `clientId` here? If we don't use it we should not store it I guess.